code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/notification/builders/ManualCarouselNotificationBuilder.kt

```diff
@@ -215,13 +213,14 @@ internal object ManualCarouselNotificationBuilder {
                     )
                 } else {
                     Triple(
+                        imageUris.size - 1,
```

Comment:
Doe we check if this is >0 in the calling method somewhere?

Reply:
we verify that we have at least 3 images before calculating indices so this is guaranteed to be greater than 0.